src/app/lib/utilities/preprocessor/rules/filterContentType.js

```diff
@@ -1,7 +1,7 @@
 import pathOr from 'ramda/src/pathOr';
 
 const whitelist = ['STY', 'MAP', 'PGL', 'LIV', 'PRO'];
-const contentTypes = ['Text', 'Feature', 'Video', 'Audio'];
+const contentTypes = ['Text', 'Feature', 'Audio', 'Video', 'Gallery'];
```

Comment:
```suggestion
const contentTypes = ['Text', 'Video', 'Audio', 'Gallery', 'Feature'];
```
Maybe we could have these types in a similar order to the whitelist above? Just a personal preference though

Reply:
Mmm the `contentTypes` are all PRO types, so I would keep them like that.